mzLib/FlashLFQ/IsoTracker/SearchTarget.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Omics.Modifications;
+
+namespace FlashLFQ.IsoTracker
+{
+    public class SearchTarget
+    {
+        public List<ModificationMotif> TargetMotifs; // motif string list
+        public List<Regex> TargetMotifPattern; // motif regex list for sequence checking
+
+        /// <summary>
+        /// Directly use the motif string to create a SearchingTarget
+        /// </summary>
+        /// <param name="motifs"></param>
+        /// <param name="option2"></param>
+        public SearchTarget(List<char> motifs)
+        {
+            if (motifs == null) return; // temporary fix for null motif list, if the user does not set the motif list, then no filter will be applied
```

Comment:
Set TargetMotifs to new list before returning on null char list. This ensure that TargetMotifs will never be null